datafusion/physical-optimizer/src/optimizer.rs

```diff
@@ -137,6 +138,7 @@ impl PhysicalOptimizer {
             // are not present, the load of executors such as join or union will be
             // reduced by narrowing their input tables.
             Arc::new(ProjectionPushdown::new()),
+            Arc::new(WrapLeaves::new()),
```

Comment:
Can we please call this pass something related to Cancel or Yield? Like `InsertYieldExec` to make it clearer what it is doing?

Reply:
Addressed in latest PR.